src/Generators/Microsoft.Gen.MetricsReports/MetricDefinitionEmitter.cs

```diff
@@ -86,56 +110,62 @@ private void GenMetricMethodDefinition(ReportedMetricMethod metricMethod, Cancel
                 try
                 {
                     cancellationToken.ThrowIfCancellationRequested();
+                    OutLn("{");
 
-                    OutLn("    {");
-
-                    OutLn($"     \"MetricName\": \"{metricMethod.MetricName.Replace("\\", "\\\\").Replace("\"", "\\\"")}\",");
+                    OutLn($"    \"MetricName\": \"{metricMethod.MetricName.Replace("\\", "\\\\").Replace("\"", "\\\"")}\",");
```

Comment:
```suggestion
                    OutIndent();
                    OutLn($"\"MetricName\": \"{metricMethod.MetricName.Replace("\\", "\\\\").Replace("\"", "\\\"")}\",");
```

ditto to remove spaces in OutLn following this indentation.